src/cephadm/cephadm.py

```diff
@@ -4027,20 +4027,22 @@ def install_base_units(ctx, fsid):
         first child (bash), but that isn't the ceph daemon.  This is simpler and
         should be harmless.
         """
+        targets: List[str] = ['ceph-mon', 'ceph-mgr', 'ceph-mds', 'ceph-osd', 'ceph-fuse',
+                              'radosgw', 'rbd-mirror', 'cephfs-mirror', 'tcmu-runner']
```

Comment:
Super-nitpicky but: I want to encourage a one-line-per-item style like:
```suggestion
        targets: List[str] = [
            'ceph-mon',
            'ceph-mgr',
            'ceph-mds',
            'ceph-osd',
            'ceph-fuse',
            'radosgw',
            'rbd-mirror',
            'cephfs-mirror',
            'tcmu-runner',
        ]
```

It's easier to read (IMO) and it creates less churn in diffs if you need to add/remove/etc items from the list. Plus it matches formatting that a tool like 'black' produces.

Reply:
updated to this format